psi4/src/psi4/psimrcc/operation_compute.cc

```diff
@@ -205,8 +205,8 @@ void CCOperation::zero_target_block(int h) {
 void CCOperation::fail_to_compute() {
     outfile->Printf("\n\nSolve couldn't perform the operation ");
     print_operation();
-
-    exit(EXIT_FAILURE);
+    outfile->Flush();
```

Comment:
Why do you `flush` some times but not others? I assume (perhaps naively) that information would all get flushed before the program actually terminates, even without that line.

Reply:
I've found through experience that all information may not get flushed to output in C++ when an exception is thrown. I've added flushes wherever there's a danger of pertinent output not being printed out. Does Psi4 catch exceptions and flush the output there?

Reply:
Not that I've ever seen.